jinahub/indexers/searcher/FaissSearcher/faiss_searcher.py

```diff
@@ -610,9 +613,8 @@ def _add_delta(self, delta: Generator[Tuple[str, bytes, datetime], None, None]):
             if idx is None:  # add new item
                 if vec_buffer is None:
                     continue
-                vec = np.frombuffer(vec_buffer, dtype=np.float32).reshape(
-                    1, -1
-                )  # shape [1, D]
+                # shape [1, D]
+                vec = np.frombuffer(vec_buffer).reshape(1, -1).astype(np.float32)
```

Comment:
I would prefer having PSQL directly store the right vectors data type and not do the type conversion here. Reasoning: This will shift time spend towards *indexing* time and not *syncing* time.

Reply:
FaissSearcher could still receive this data from other sources. We still need it in there. I will make the default PSQL dtype be f32

Reply:
Fair enough.